test/support/setup_helpers.ex

```diff
@@ -0,0 +1,16 @@
+defmodule App.SetupHelpers do
+  use Phoenix.ConnTest
+  import AppWeb.Router.Helpers
+
+  @endpoint AppWeb.Endpoint
+
+  def person_login(_) do
```

Comment:
this function seems self-explanatory to us _now_, but it could use a comment for the future. 😉